pkg/inclusion/paths_test.go

```diff
@@ -0,0 +1,229 @@
+package inclusion
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_calculateSubTreeRootCoordinates(t *testing.T) {
```

Comment:
👍

Reply:
[for other reviewers] I had a tough time verifying the 8 leaf tests but drawing it out helped me so leaving here in case helpful

![03CD7E57-681D-498F-8959-A2D46D2FA9A0_1_102_a](https://user-images.githubusercontent.com/3699047/185482318-b26fbbac-ae59-4ae5-b971-b70da8e90f9b.jpeg)

Reply:
yes! thank you!!

need an ascii version of this in the docs

Reply:
yes ser

```ascii
Depth       Position
0              0
              / \
             /   \
1           0     1
           /\     /\
2         0  1   2  3
         /\  /\ /\  /\ 
3       0 1 2 3 4 5 6 7
```